parity/main.rs

```diff
@@ -85,17 +85,32 @@ mod sync;
 mod snapshot;
 
 use std::{process, env};
+use std::io::prelude::*;
+use std::fs::File;
+use util::sha3::Hashable;
 use cli::print_version;
 use configuration::{Cmd, Configuration};
 use deprecated::find_deprecated;
 
+fn print_hash_of(maybe_file: Option<String>) -> Result<String, String> {
+	if let Some(file) = maybe_file {
+		let mut f = try!(File::open(&file).map_err(|_| "Unable to open file".to_owned()));
+		let mut bytes = vec![];
+		try!(f.read_to_end(&mut bytes).map_err(|_| "Unable to read from file".to_owned()));
+		Ok(bytes.sha3().hex())
```

Comment:
I think it's sensible to use the function created by @tomusdrw in other pr (once it's merged) https://github.com/ethcore/parity/pull/1993/files#diff-ce115bfe675e73b03323a5a2d6d2ab84R70